website_hr_job_unpublication/data/ir_cron.xml

```diff
@@ -0,0 +1,20 @@
+<?xml version="1.0" encoding="utf-8" ?>
+<!-- Copyright 2017 ACSONE SA/NV
+     License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl). -->
+<odoo>
+
+    <record id="ir_cron_hr_job_auto_unpublish" model="ir.cron">
+        <field name="name">Hr Job Automatic Unpublication</field>
+        <field name="user_id" ref="base.user_root" />
+        <field name="active" eval="True"/>
+        <field name="interval_type">hours</field>
+        <field name="numbercall">-1</field>
+        <field name="doall" eval="False" />
+        <field name="nextcall"
+            eval="(DateTime.today()).strftime('%Y-%m-%d %H:%M:%S')" />
```

Comment:
Is the parentheses necessary?

Reply:
@tarteo I think yes

Reply:
@GillesTephaneMeyomesse I think it should work without parenthesis